docs/en/ingest-management/elastic-agent/configuration/processors/processor-registered_domain.asciidoc

```diff
@@ -0,0 +1,42 @@
+[[registered_domain-processor]]
+== Registered Domain
+
+++++
+<titleabbrev>registered_domain</titleabbrev>
+++++
+
+The `registered_domain` processor reads a field containing a hostname and then
+writes the "registered domain" contained in the hostname to the target field.
+For example, given `www.google.co.uk` the processor would output `google.co.uk`.
+In other words the "registered domain" is the effective top-level domain
+(`co.uk`) plus one level (`google`). Optionally, it can store the rest of the
+domain, the `subdomain` into another target field.
```

Comment:
```suggestion
In other words, the "registered domain" is the effective top-level domain
(`co.uk`) plus one level (`google`). Optionally, it can store the rest of the
domain, i.e. the `subdomain`, into another target field.
```